### PR TITLE
LPD-2980 The delete option should be disable in the management toolbar when there is only one layout left

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-29801)

### Changes

Don't sho the delete option in the management tollbar when there is only one layout left by using the `isShowDeleteAction` method.

## Test scenario 

1. Create a page (only one)
2. Try to delete it from the management toolbar

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-admin-web-test/src/testIntegration/java/com/liferay/layout/admin/web/internal/display/context/test/LayoutsAdminDisplayContextTest.java [hunk 1]
```diff
@@ -62,6 +71,41 @@ public void setUp() throws Exception {
...
+
+		Group group = _groupLocalService.getGroup(
+			company.getCompanyId(), GroupConstants.GUEST);
+
+		_layoutLocalService.deleteLayouts(
+			group.getGroupId(), false,
+			ServiceContextTestUtil.getServiceContext(
+				_group, TestPropsValues.getUserId()));
+
+		Layout layout1 = LayoutTestUtil.addTypePortletLayout(
```
Comment (ealonso) [inline modules/apps/layout/layout-admin-web-test/src/testIntegration/java/com/liferay/layout/admin/web/internal/display/context/test/LayoutsAdminDisplayContextTest.java:86]: @BarbaraCabrera you can just use the method without  LayoutConstants.DEFAULT_PLID

File: modules/apps/layout/layout-admin-web-test/src/testIntegration/java/com/liferay/layout/admin/web/internal/display/context/test/LayoutsAdminDisplayContextTest.java [hunk 2]
```diff
@@ -62,6 +71,41 @@ public void setUp() throws Exception {
...
+			group, LayoutConstants.DEFAULT_PLID);
+
+		Layout layout2 = LayoutTestUtil.addTypeContentLayout(
+			group, LayoutConstants.DEFAULT_PLID);
+
+		List<String> availableActions = _getAvailableActions(group, layout2);
+
+		Assert.assertFalse(availableActions.contains("convertSelectedPages"));
```
Comment (ealonso) [inline modules/apps/layout/layout-admin-web-test/src/testIntegration/java/com/liferay/layout/admin/web/internal/display/context/test/LayoutsAdminDisplayContextTest.java:94]: @BarbaraCabrera Could you please alphabetic order the contains?

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**